netmap/netmap_contract.go

```diff
@@ -190,9 +190,9 @@ func UpdateInnerRing(keys []interop.PublicKey) {
 	common.SetSerialized(ctx, innerRingKey, irList)
 }
 
-// Register method tries to add new candidate to the network map by
-// emitting AddPeer notification. Should be invoked by the registree.
-func Register(nodeInfo []byte) {
+// AddPeerIR method tries to add new candidate to the network map.
+// Should be invoked in notary-enabled environment by the alphabet.
```

Comment:
add somehow info that this should be invoked "*only* in notary-enabled...", please

Reply:
i meant that it could be read as "Should be invoked ... by the alphabet", not clear that it is prohibited to use in notary disabled environment at all

Reply:
Fixed.